docs/querying_history.rst

```diff
@@ -98,16 +98,42 @@ This will change the ``poll`` instance to have the data from the
 as_of
 -----
 
-This method will return an instance of the model as it would have existed at
-the provided date and time.
+The HistoryManager allows you to query a point in time for the latest historical
+records or instances.  The ``as_of`` method will return instances from a specific
+date and time that you specify, and return a queryset that you can use to further
+filter the result.
```

Comment:
Doesn't `as_of` either return a single instance or a queryset? If so, I think this should be reworded a little to make that clear.

> The ``as_of`` method will return an instance or a queryset from a specified date and time.

Ideally we'd call out when it will return a instance vs queryset too.

Reply:
If you explore the history of the related issues, you will see most people wanted to leave as_of alone and use something else to represent a proper timepoint query.  I agree with you that as_of returning an instance or a queryset is confusing.  Previously it returned either an instance or an array of instances.